docs/database/azure-cosmos-db-component.md

```diff
@@ -72,21 +72,14 @@ In your app host project, register the .NET Aspire Azure Cosmos DB component and
 
 ```csharp
 // Service registration
-var cosmosdb = builder.ExecutionContext.IsPublishMode
-    ? builder.AddAzureCosmosDB("cdb")
-    : builder.AddConnectionString("cdb");
+var cosmos = builder.AddAzureCosmosDB("cosmos");
+var cosmosdb = cosmos.AddDatabase("cosmosdb");
```

Comment:
Let's chain these together to avoid an unnecessary variable. And let's double check with @davidfowl - as I thought we wanted to show the `.IsPublishMode` ternary bits here.

```suggestion
var cosmosdb = builder.AddAzureCosmosDB("cosmos")
                      .AddDatabase("cosmosdb");
```